include/TrustWalletCore/TWCoinType.h

```diff
@@ -109,11 +109,12 @@ enum TWCoinType {
     TWCoinTypeEvmos = 10009001,
     TWCoinTypeNativeEvmos = 20009001,
     TWCoinTypeMoonriver = 10001285,
-    TWCoinTypeMoonbeam = 10001284,
+    TWCoinTypeMoonbeam = 10001284,    
```

Comment:
(Minor) some extra trailing spaces added, should be remove

Reply:
Hi @catenocrypt we are not sure how these 4 whitespaces appeared here. We've removed it.